src/cljs/org/broadinstitute/firecloud_ui/page/workspace/summary/tab.cljs

```diff
@@ -37,7 +38,8 @@
                    :dismiss-self #(swap! state dissoc :editing-acl?)
                    :update-owners #(swap! state update-in [:server-response :workspace] assoc "owners" %)}])
      (when (:cloning? @state)
-       [WorkspaceCloner {:dismiss #(swap! state dissoc :cloning?)
+       [WorkspaceCloner {:nav-context nav-context
+                         :dismiss #(swap! state dissoc :cloning?)
```

Comment:
I'll change :dismiss to :on-success and have it both clear the box from the screen and use the nav-context to navigate to the new clone workspace.